test/window-test.js

```diff
@@ -34,6 +34,10 @@ describe("Window", () => {
     it("has FormData", () => {
       expect(window).to.have.property("FormData");
     });
+
+    it("has MutationObserver", () => {
+      expect(window).to.have.property("MutationObserver").that.equal(MutationObserver);
```

Comment:
isn't it overkill to check value? actual functionality is confirmed by other test

```suggestion
      expect(window).to.have.property("MutationObserver");
```

Reply:
TDD forced me...